projects/openapi-cli/src/specs/patches/generators/missing-path.ts

```diff
@@ -41,6 +41,9 @@ export function* missingPathPatches(
               in: 'path',
               name: parameterName,
               required: true,
+              schema: {
```

Comment:
fix reported by @philsturgeon who noticed that these were being added without schemas and failing spectral's oas checks.